slip-0044.md

```diff
@@ -643,9 +643,9 @@ Coin type | Path component (`coin_type'`) | Symbol | Coin
 612   | 0x80000264 |        | [Calamari Network Private Asset](https://calamari.manta.network/)
 613   | 0x80000265 | WCN    | [Widecoin](https://Widecoin.org/)
 614   | 0x80000266 |        |
-615   | 0x80000267 |        |
-616   | 0x80000268 |        |
-617   | 0x80000269 |        |
+615   | 0x80000267 | PSWAP  | [PolkaSwap](https://sora.org/soratokens) [https://polkaswap.io]
```

Comment:
the last element is not a valid markdown syntax

Reply:
Corrected.